src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* [Breaking Change] Output type for -Type ConnectionStrings is changed to List<DatabaseAccountConnectionString> to expose keyKind and type fields
```

Comment:
Hi @carjackson-msft, it's not a breaking now aftrer your update. Could you update the changlog line to remove "Breaking Change"?

Reply:
And please update the line to more user-friendly.